@vates/parse-duration/parse-duration.test.mjs

```diff
@@ -0,0 +1,22 @@
+import { describe, it } from 'node:test'
```

Comment:
I don't see any changes in `package.json`, how are these tests run?

Reply:
please add a test command in the script section of this package. Then ensure theses tests are run by the CI